_includes/layouts/default.html

```diff
@@ -11,25 +11,17 @@
       {% endif %}
     {% endcomment %}
   </head>
-  <body class="{{ layout.class }} {{ page.class }} {% if site.site_width %}site-{{ site.site_width }}{% endif %}">
-
+  <body class="{{ layout.class }} {{ page.class }}">
     {% include "skipnav.html" %}
     {% include "header.html" %}
 
-    {% if page.tagline and page.intro %}
-    <section class="grid-container usa-section">
-      <div class="grid-row grid-gap">
-        <div class="tablet:grid-col-4">
-          <h2 class="font-heading-xl margin-top-0 tablet:margin-bottom-0">{{ page.tagline }}</h2>
-        </div>
-        <div class="tablet:grid-col-8 usa-prose">
-          {{ page.intro | markdownify }}
-        </div>
-      </div>
-    </section>
-    {% endif %}
+    {% comment %}
+      code smell: Altering layouts based on the value of `layout`.
```

Comment:
😅 😅 😅 same here? -- do we have an issue to track this? is some deeper thought about layout required here?

Reply:
We do! The last task in [the epic](https://github.com/18F/18f.gsa.gov/issues/3839) is to clean up todos and code smells.